package.json

```diff
@@ -7,6 +7,9 @@
   "files": [
     "dist"
   ],
+  "config": {
+    "knex": "--knexfile ./config/knexfile.ts"
+  },
   "scripts": {
     "audit:resolve": "SHELL=sh resolve-audit",
```

Comment:
Btw, just happened to see this, but you can add `--production` to both `audit:resolve` and `audit:check` to avoid having ~42k lines of `audit-resolve.json`.

Reply:
Thanks for the suggestion @stevenwjy. @lewisdaly @eoln Any thoughts on this?